Releases/Boston/README.md

```diff
@@ -0,0 +1,16 @@
+# Skuid Boston
+
+Resources to guide you through the features that come with Skuid Boston release (v13.X), including a demo page and an accompanying design system. 
```

Comment:
Would it be better to use lower case "x". like:  Boston release (v13.x)

Reply:
Aye, I'd lean toward a lowercase `x`, but either is fine.